templates/base/packages/nextjs/.env.example.template.mjs

```diff
@@ -14,7 +14,7 @@ const contents = ({ additionalVars }) => {
 # More info: https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables
 NEXT_PUBLIC_ALCHEMY_API_KEY=
 NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID=
-${additionalVars.join("\n")}
+${additionalVars[0].join("\n")}
```

Comment:
We should add check : 

```js
${additionalVars[0] ? additionalVars[0]?.join("\n") : ''}
```

Other wise it fails in normal `yarn cli --skip` command. 

![Screenshot 2025-01-01 at 3 02 22 PM](https://github.com/user-attachments/assets/4d4d22cb-fa5f-4620-bb77-507b4bdc890a)